nexus/db-queries/src/db/queries/sled_reservation.rs

```diff
@@ -281,13 +308,24 @@ pub fn sled_insert_resource_query(
             ).param().sql(" <= sled.usable_physical_ram AND
                 COALESCE(SUM(CAST(sled_resource_vmm.reservoir_ram AS INT8)), 0) + "
             ).param().sql(" <= sled.reservoir_size
-        ),");
+        ),")
+        .bind::<sql_types::Uuid, _>(resource.sled_id.into_untyped_uuid())
+        .bind::<sql_types::BigInt, _>(resource.resources.hardware_threads)
+        .bind::<sql_types::BigInt, _>(resource.resources.rss_ram)
+        .bind::<sql_types::BigInt, _>(resource.resources.reservoir_ram);
 
     // "our_aa_groups": All the anti-affinity group_ids to which our instance belongs.
     subquery_our_aa_groups(&mut query);
+    query.bind::<sql_types::Uuid, _>(
+        resource.instance_id.unwrap().into_untyped_uuid(),
```

Comment:
take it or leave it, not a big deal: couldn't we do `let instance_id = resource.instance_id.unwrap().into_untyped_uuid()` once and use that for every bind param?

Reply:
ab9850fad, especially since it contributed to such long lines